feature/RZA-250134 : Login, DashBaord, Signup,Forgot password pages added

Ayu-Ty opened the pull request:
# Description:

1. Added signup page. under `/login`
<img width="1625" height="870" alt="Capture" src="https://github.com/user-attachments/assets/51ff524e-599f-4382-8feb-b6ddc4c1faef" />

Ayu-Ty committed:
added signup

Ayu-Ty committed:
Merge branch 'develop' into feature/RZA-250134

Ayu-Ty committed:
lintify

Ayu-Ty committed:
lintify

Ayu-Ty committed:
Merge branch 'develop' into feature/RZA-250134

Ayu-Ty committed:
Merge branch 'develop' into feature/RZA-250134

Ayu-Ty committed:
added login page

Ayu-Ty committed:
added light mode

Ayu-Ty committed:
added light mode

recursivezero reviewed (changes requested):
change the images of signup   Add 
1. Story page in light mode
2. India Map
3. Hindi Varnmala page 

also change the folder name to **screenshots** 

Add Forgot Password page too.

Ayu-Ty committed:
Merge branch 'develop' into feature/RZA-250134

Ayu-Ty committed:
added dashbaord, forgot password

Ayu-Ty commented:
1. Added new images to the slider.
2. Added dashboard page
3. Added forgot password page
4. Changed the name of folder to screenshots
<img width="1580" height="853" alt="image" src="https://github.com/user-attachments/assets/dc4f24dd-4f27-4b87-8758-289441612c5b" />

Ayu-Ty committed:
lintify

recursivezero commented:
<img width="3745" height="1962" alt="image" src="https://github.com/user-attachments/assets/b77d0a40-bb0a-4974-90af-6465b4b051a1" />
 make it responsive

Ayu-Ty committed:
fixes

Ayu-Ty committed:
lintify

Ayu-Ty commented:
1.  Added responsiveness
2.  Removed image slider from forgot password page
3. Moved back to website button left container
4.  Added header to dashboard 

<img width="1604" height="863" alt="image" src="https://github.com/user-attachments/assets/6b7fbbc5-29bd-4ff8-b95c-eca8dcf1fbe9" />

Ayu-Ty committed:
Merge branch 'develop' into feature/RZA-250134

xkeshav commented:
- password field: when click on eye icon then icon should changed as crossed over eye so that user can identify .
- use `<BaseLayout sidebar={false} />` to include header and footer in allpages
- No duplicate `<head>`, `< body>` , `<main>` tag on any page 
- make all pages fullwidth 
- take screenshot of the pages not the header one 
- Add Panel page and remove Draw page screenshot
- forgot password page make input box width more

Ayu-Ty committed:
added baselayout

chhaviS04 committed:
Added my blog

Ayu-Ty committed:
added baselayout to forgot-password

Ayu-Ty committed:
fixed image bug

chhaviS04 committed:
Removed my blog from this branch

recursivezero committed:
header logo link wrapped; playground layout changed for max-width

Ayu-Ty committed:
added images, reduced forgot password's width

Ayu-Ty committed:
lintify

Ayu-Ty committed:
lintify

recursivezero committed:
update gitignore to prevent node_modules from  any folder

xkeshav committed:
Merge branch 'develop' into feature/RZA-250134

recursivezero committed:
[RZA-250134]: survey page selected  option and question css fixed

recursivezero committed:
[RZA-250134]: css fix for signup images

recursivezero committed:
[RZA-250134]: lint fix
